src/app/models/AssociationDecorators.ts

```diff
@@ -165,17 +164,22 @@ function createModelDecorator<M extends AbstractModel, S>(
 
     // Create service and request from API
     const service = injector.get(serviceToken.token);
-    const request = createRequest(service, identifier, parameters).pipe(
-      map((response) => {
-        // Change cached value to BehaviorSubject so that it will instantly return
-        updateCache(target, cachedModelKey, new BehaviorSubject(response));
-        return response;
-      })
+    createRequest(service, identifier, parameters).subscribe(
+      (model) => updateCache(target, cachedModelKey, model),
```

Comment:
Ok, this is great, however, if the observeable emits more than once we'll only get the last value.

Why might it omit more than once? Say we want the first two pages of API results to load by default for all associations.

Also, `updateCache` should probably be renamed to `updateBackingField`

Reply:
So if the response is in array, append to the array. What about single responses? Just override still?

Reply:
Yeah, override. I can't think of any other reasonable behaviour